modules/Makefile.kubernetes

```diff
@@ -26,6 +26,9 @@ endef
 define kubectl_apply
 	$(call assert,CLUSTER_NAMESPACE)
 	$(call assert,CLUSTER_DOMAIN)
+	# Optionally load parameters for the kubernetes resources
+	-@source $(KUBERNETES_RESOURCE_PATH)/default.k8sparameters
```

Comment:
Move the config into a subdirectory such as "config/default.env" and "config/$CLUSTER.env"